src/components/Dropdown/styles.ts

```diff
@@ -90,3 +91,8 @@ export const List = styled.ul<ListProps>`
     }
   }
 `;
+
+export const Button = styled(CustomButton)`
```

Comment:
The old `Button` from `Dropdown/index.stories.tsx` was based on the `CustomButton` component and not only on its style.

Is this what you want?

Reply:
CustomButton with variant 'link' does not accept children that are just text, in this case an icon is needed after the text of the button. It was the way I found to solve for now